packages/expo/src/winter/fetch/FetchResponse.ts

```diff
@@ -81,4 +90,10 @@ export class FetchResponse extends ConcreteNativeResponse implements Response {
   clone(): FetchResponse {
     throw new Error('Not implemented');
   }
+
+  private removeAllRegisteredListeners() {
+    this.removeAllListeners('didReceiveResponseData');
+    this.removeAllListeners('didComplete');
+    this.removeAllListeners('didFailWithError');
```

Comment:
Would be great if our EventEmitter supports `removeAllListeners` without any arguments to remove listeners for all events